lib/cmds/issue.js

```diff
@@ -68,6 +70,7 @@ Issue.DETAILS = {
     },
     shorthands: {
         'a': ['--all'],
+        'as': ['--assign'],
```

Comment:
@henvic I wasn't sure what the best shorthand would be, so this was just my placeholder unless you think it's the best option.